core/capabilities/targets/write_target_test.go

```diff
@@ -82,36 +115,11 @@ func TestWriteTarget(t *testing.T) {
 		require.NotNil(t, response)
 	})
 
-	t.Run("succeeds with empty report", func(t *testing.T) {
```

Comment:
Removed because we no longer support a use case in which an empty report is passed to the target capability.